lib/advanced_data_grid.dart

```diff
@@ -436,6 +473,24 @@ class _DataGridState extends State<DataGrid> {
         ],
       );
 
+      final tooltipMessages = {
+        'Data': 'Total amount of data included',
+        'Local Calls': 'Total minutes of calls included',
+        'Local SMS': 'Total SMS messages included',
+        'Incl. Number': 'Whether the bundle includes a local phone number',
+      };
```

Comment:
Not sure we need this object to define the strings for the tooltips. 

As we are passing the tooltip message into the DataColumn, we can use the string entered there directly instead of checking it against the array above.

Other than that, looks good! 👍